PryntTrimmerView/Classes/Trimmer/PryntTrimmerView.swift

```diff
@@ -172,214 +258,253 @@ public protocol TrimmerViewDelegate: AVAssetTimeSelectorDelegate {
+        leftHandleLabel.centerXAnchor.constraint(equalTo: leftHandleView.centerXAnchor, constant: -15).isActive = true
+        leftHandleLabel.isHidden = true
 
-        leftHandleKnob.heightAnchor.constraint(equalTo: heightAnchor, multiplier: 0.5).isActive = true
-        leftHandleKnob.widthAnchor.constraint(equalToConstant: 2).isActive = true
+        leftHandleKnob.heightAnchor.constraint(equalToConstant: 24).isActive = true
+        leftHandleKnob.widthAnchor.constraint(equalToConstant: 8).isActive = true
         leftHandleKnob.centerYAnchor.constraint(equalTo: leftHandleView.centerYAnchor).isActive = true
         leftHandleKnob.centerXAnchor.constraint(equalTo: leftHandleView.centerXAnchor).isActive = true
 
         rightHandleView.isUserInteractionEnabled = true
-        rightHandleView.layer.cornerRadius = 2.0
+        rightHandleView.layer.cornerRadius = 8.0
+        if #available(iOS 11.0, *) {
+            rightHandleView.layer.maskedCorners = [.layerMaxXMinYCorner, .layerMaxXMaxYCorner]
+        }
         rightHandleView.translatesAutoresizingMaskIntoConstraints = false
         addSubview(rightHandleView)
 
         rightHandleView.heightAnchor.constraint(equalTo: heightAnchor).isActive = true
         rightHandleView.widthAnchor.constraint(equalToConstant: handleWidth).isActive = true
-        rightConstraint = rightHandleView.leftAnchor.constraint(equalTo: rightAnchor)
-        rightConstraint?.isActive = true
+        rightHandleConstraint = rightHandleView.leftAnchor.constraint(equalTo: assetPreview.rightAnchor)
+        rightHandleConstraint?.isActive = true
         rightHandleView.centerYAnchor.constraint(equalTo: centerYAnchor).isActive = true
 
         rightHandleKnob.translatesAutoresizingMaskIntoConstraints = false
+        rightHandleKnob.image = UIImage(named:"handleRightArrow")
         rightHandleView.addSubview(rightHandleKnob)
 
-        rightHandleLabel.layer.backgroundColor = UIColor.darkGray.withAlphaComponent(0.8).cgColor
-        rightHandleLabel.textColor = UIColor.white
-        rightHandleLabel.textAlignment = .center
-        rightHandleLabel.text = "00:00:00 AM"
-        rightHandleLabel.frame = CGRect(x: -60, y: -40, width: 120.0, height: 30.0)
-        rightHandleLabel.layer.cornerRadius = 5.0
+        rightHandleLabel.translatesAutoresizingMaskIntoConstraints = false
         rightHandleView.addSubview(rightHandleLabel)
+        rightHandleLabel.bottomAnchor.constraint(equalTo: rightHandleView.topAnchor, constant: -40).isActive = true
+        rightHandleLabel.centerXAnchor.constraint(equalTo: rightHandleView.centerXAnchor, constant: -24).isActive = true
+        rightHandleLabel.isHidden = true
 
-        rightHandleKnob.heightAnchor.constraint(equalTo: heightAnchor, multiplier: 0.5).isActive = true
-        rightHandleKnob.widthAnchor.constraint(equalToConstant: 2).isActive = true
+        rightHandleKnob.heightAnchor.constraint(equalToConstant: 24).isActive = true
```

Comment:
consts please :) Also I see a lot of 24s here - if the height is the same for a few components please make their height equal each other, not set each individually to 24.

For example, the strip should be constraint to 24 const + the handles height should be equal to the strip height